test/e2e/network/pod_lifecycle.go

```diff
@@ -81,19 +83,18 @@ var _ = common.SIGDescribe("Connectivity Pod Lifecycle", func() {
 	})
 
 	ginkgo.It("should be able to connect to other Pod from a terminating Pod", func(ctx context.Context) {
-		ginkgo.By("Creating 1 webserver pod able to serve traffic during the grace period of 300 seconds")
-		gracePeriod := int64(100)
-		webserverPod := e2epod.NewAgnhostPod(ns, "webserver-pod", nil, nil, nil, "netexec", "--http-port=80", fmt.Sprintf("--delay-shutdown=%d", gracePeriod))
+		ginkgo.By("Creating 1 webserver pod able to serve traffic during the grace period of " + strconv.FormatInt(longNetworkGracePeriod, 10) + " seconds")
+		webserverPod := e2epod.NewAgnhostPod(ns, "webserver-pod", nil, nil, nil, "netexec", fmt.Sprintf("--http-port=%d --delay-shutdown=%d", httpPort, longNetworkGracePeriod))
```

Comment:
this has to be comma separated, do not aggregate them

Reply:
ok, I thought fmt.Sprintf would make it slower two times. as aggregation seems faster IMHO. thanks for the feedback, done!